src/utils.js

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+
+const baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+
+export async function createGame(user, score) {
+  const requestBody = {
+    user,
+    score,
+  };
+  await axios.post(`${baseUrl}5H3ZhzHBptAjLZzd7hXU/scores`, requestBody);
+}
+
+export async function getData() {
+  const response = await axios.get(`${baseUrl}5H3ZhzHBptAjLZzd7hXU/scores`);
+  return response.data;
+}
```

Comment:
- [ ] Let's make sure to use Arrow Functions as these are a new standard introduced on ES6 that it's taking over the old function syntax. Since arrow functions cannot be directly exported, we will have to declare our exports at the end of the file.

Reply:
Sure, I will definitely clean things up.